javascript/extractor/src/com/semmle/js/extractor/Main.java

```diff
@@ -41,7 +41,7 @@
 	 * such a way that it may produce different tuples for the same file under the same
 	 * {@link ExtractorConfig}.
 	 */
-	public static final String EXTRACTOR_VERSION = "2018-12-19";
+	public static final String EXTRACTOR_VERSION = "2019-09-01";
```

Comment:
This should have been `2019-01-09` for consistency, but it's just a tag, so doesn't matter.